File: University/ZarbeMatrixHaS1.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        byte row1 = input.nextByte();
        byte column = input.nextByte();
        byte column2 = input.nextByte();

        int [][]array1 = new int[101][101];
        int [][]array2 = new int[101][101];
        int [][]result = new int[101][101];

        for ( byte i=0 ; i < row1 ; i++){
            for ( byte j=0 ; j < column ; j++){
                array1[i][j] = input.nextInt();
            }
        }
        for ( byte i=0 ; i < column ; i++){
            for ( byte j=0 ; j < column2 ; j++){
                array2[i][j] = input.nextInt();
            }
        }

        for (int i=0 ; i < row1 ; i++){
            for (int j=0 ; j < column2 ; j++){
                for (int k=0 ; k < column ; k++){
                    result[i][j] += array1[i][k] * array2[k][j];
                }
            }
        }
        for ( byte i=0 ; i < row1 ; i++){
            for ( byte j=0 ; j < column2 ; j++){
                System.out.print(result[i][j]+" ");
            }
            System.out.println("");
        }
    }
}

File: University/ZarbeMatrixHaS1.rb
class Main
    def self.main()
        input =  java.util.Scanner
        row1 = input.nextByte()
        column = input.nextByte()
        column2 = input.nextByte()
        array1 = Array.new(101){Array.new(101){0}}
        array2 = Array.new(101){Array.new(101){0}}
        result = Array.new(101){Array.new(101){0}}
        i = 0
        while (i < row1)
            j = 0
            while (j < column)
                array1[i][j] = gets.chomp
                j += 1
            end
            i += 1
        end
        i = 0
        while (i < column)
            j = 0
            while (j < column2)
                array2[i][j] = gets.chomp
                j += 1
            end
            i += 1
        end
        i = 0
        while (i < row1)
            j = 0
            while (j < column2)
                k = 0
                while (k < column)
                    result[i][j] += array1[i][k] * array2[k][j]
                    k += 1
                end
                j += 1
            end
            i += 1
        end
        i = 0
        while (i < row1)
            j = 0
            while (j < column2)
                print(result[i][j].to_s + " ")
                j += 1
            end
            print("","\n")
            i += 1
        end
    end
end
Main.main()
